tests/puppeteer/campaigns/functional/BO/02_orders/01_orders/06_bulkUpdateOrdersStatus.js

```diff
@@ -0,0 +1,148 @@
+        await this.pageObjects.productPage.addProductToTheCart();
+        // Proceed to checkout the shopping cart
+        await this.pageObjects.cartPage.clickOnProceedToCheckout();
+        // Address step - Go to delivery step
+        const isStepAddressComplete = await this.pageObjects.checkoutPage.goToDeliveryStep();
+        await expect(isStepAddressComplete, 'Step Address is not complete').to.be.true;
+        // Delivery step - Go to payment step
+        const isStepDeliveryComplete = await this.pageObjects.checkoutPage.goToPaymentStep();
+        await expect(isStepDeliveryComplete, 'Step Address is not complete').to.be.true;
+        // Payment step - Choose payment step
+        await this.pageObjects.checkoutPage.choosePaymentAndOrder(PaymentMethods.wirePayment.moduleName);
+        const cardTitle = await this.pageObjects.orderConfirmationPage.getOrderConfirmationCardTitle();
+        // Check the confirmation message
+        await expect(cardTitle).to.contains(this.pageObjects.orderConfirmationPage.orderConfirmationCardTitle);
+      });
+    });
+
+    it('should sign out from FO', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'sighOutFO', baseContext);
+      await this.pageObjects.orderConfirmationPage.logout();
+      const isCustomerConnected = await this.pageObjects.orderConfirmationPage.isCustomerConnected();
+      await expect(isCustomerConnected, 'Customer is not connected').to.be.false;
+    });
+  });
+
+  describe('Update orders status in BO', async () => {
+    // Login into BO
+    loginCommon.loginBO();
+
+    it('should go to the orders page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToOrdersPage', baseContext);
+      await this.pageObjects.boBasePage.goToSubMenu(
+        this.pageObjects.boBasePage.ordersParentLink,
+        this.pageObjects.boBasePage.ordersLink,
+      );
+      const pageTitle = await this.pageObjects.ordersPage.getPageTitle();
+      await expect(pageTitle).to.contains(this.pageObjects.ordersPage.pageTitle);
+    });
+
+    it('should update orders status with bulk action', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'bulkUpdateOrdersStatus', baseContext);
+      const textResult = await this.pageObjects.ordersPage.bulkUpdateOrdersStatus(
+        Statuses.paymentAccepted.status,
+        false,
+        [1, 2],
+      );
+      await expect(textResult).to.equal(this.pageObjects.ordersPage.successfulUpdateMessage);
+    });
+
+    ['first', 'second'].forEach((arg, index) => {
+      it(`should check ${arg} order status`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `checkOrderStatus${index + 1}`, baseContext);
+        const orderStatus = await this.pageObjects.ordersPage.getTextColumn('osname', index + 1);
+        await expect(orderStatus, 'Order status is not correct').to.equal(Statuses.paymentAccepted.status);
```

Comment:
The label is not correct. Isn't it ?

Reply:
`Order status is not correct` is the error message that will display if the expect fails (if that what you are talking about).

Reply:
Do we have some more informations if it fails ? like " Expected waiting for payment, got waiting for delivery "

Reply:
yes, when fail, we have expected values and actual one, like this 
![image](https://user-images.githubusercontent.com/48441421/77898144-00e87680-727b-11ea-9bb8-3d3c06e3b774.png)

Reply:
That should be fine then